### PR TITLE
Add fuzzer test for capec_map_enricher.py

### PR DESCRIPTION
**Pull request by abhijit9040**
This PR adds coverage-guided fuzz testing for capec_map_enricher.py to improve robustness when handling malformed or unexpected JSON/YAML input.

Key changes:
- Added tests/scripts/capec_map_enricher_fuzzer.py using Atheris
- Mocked filesystem I/O and CLI arguments to isolate enrichment logic
- Fuzzed edge cases such as empty catalogs, malformed IDs, and varied version/edition inputs

Reference issue-#2165

### CODE REVIEW

**Comment by sydseter** [comment]
length=120 --max-complexity=10 --ignore=E203,W503
Loading .env environment variables...
./tests/scripts/capec_map_enricher_fuzzer.py:7:1: F401 'unittest.mock.MagicMock' imported but unused
./tests/scripts/capec_map_enricher_fuzzer.py:8:1: F401 'pathlib.Path' imported but unused

**Review by sydseter — CHANGES_REQUESTED** [review]
You have some style issues there.

**Comment by sydseter** [comment]
There is tje error:

  File "capec_map_enricher_fuzzer.py", line 119, in test_main
  File "capec_map_enricher_fuzzer.py", line 96, in test_main
  File "unittest/mock.py", line 1430, in __enter__
  File "pkgutil.py", line 715, in resolve_name
AttributeError: module 'scripts' has no attribute 'capec_map_enricher'

**Comment by abhijit9040** [comment]
Hi @sydseter , can you my review my pr . If any changes needed .

File: .github/workflows/build-website.yml
Comment (sydseter) [inline .github/workflows/build-website.yml:1]: What did you change here? If you believe there is an issue please open separate pull-request. Pøease revert this so that it is easier to asess the fuzzing test.

File: .github/workflows/copi-build.yml
Comment (sydseter) [inline .github/workflows/copi-build.yml:1]: Same here.  If you believe there is an issue please open separate pull-request. Pøease revert this so that it is easier to asess the fuzzing test.

File: .github/workflows/build-website.yml [hunk 1]
```diff
@@ -44,23 +44,4 @@
               pnpm install          # Install dependencies
               npm run build        # Build production version
               pnpm audit --prod
-              pnpm run coverage
-
-          - name: Smoke test
```
Comment (sydseter) [inline .github/workflows/build-website.yml:49]: It seems like this got lost in a merge? Could you just reset your commit and try to only commit the [capec_map_enricher_fuzzer.py‎](https://github.com/OWASP/cornucopia/pull/2197/changes#diff-19cccdd33de334bb8be6fed0dd9cc02b817adcc7309ee6674ce69e11f75241a3) file?

Comment (sydseter) [reply .github/workflows/build-website.yml:49]: git reset HEAD~7

**Comment by abhijit9040** [comment]
Hi @sydseter,

I’ve cleaned up the PR as requested:

- Clean history: Performed a git reset --hard and force-push. The PR now only includes tests/scripts/capec_map_enricher_fuzzer.py.
- Fuzzer update: Switched to patch.object to ensure compatibility with ClusterFuzzLite (PyInstaller) .
- ClusterFuzzLite check: The failure is due to a Docker API version issue in the GitHub runner environment, not related to the fuzzer code.

**Comment by sydseter** [comment]
@abhijit9040 Thank you so much for you time!
